templates/server/src/main/resources/archetype-resources/core/src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>   

```

Comment:
this file is only needed for liquibase. It would be great if we do not get the `changelog*.xml` files if we generate the project using `flyway`. However, I assume that maven archetype and velocity are too limited to solve this.
So OK if you do not find an easy solution.

Reply:
https://stackoverflow.com/questions/20331094/conditionally-include-exclude-files-while-creating-project-using-maven-archetype